packages/@okta/migrate-from-jekyll/index.js

```diff
@@ -221,6 +221,17 @@ function run() {
 
 
   fs.writeFileSync(docsRoot+'/.vuepress/redirects.json', JSON.stringify(redirects, null, 2))
+
+  try {
+    fs.writeFileSync(docsRoot + '/conductor.yml', 'redirects: \r\n')
+    //Build conductor redirects
+    redirects.forEach((redirect) => {
+      fs.appendFile(docsRoot + '/conductor.yml', ' - from: ' + redirect.path + '\r\n   to: ' + redirect.redirect + '\r\n')
```

Comment:
did you mean `appendFileSync`?  This won't catch errors to appends because they will be async.